### ISSUE TITLE
Does PresentMon actually support AMD products ?

### ISSUE THREAD

**Issue by Derik-for-real24** [GameTechDev/PresentMon#211]
im on AMD 6950xt with 5800x, when I open PresentMon-1.10.0.exe, nothing happens, does this mean this tool is not supported for AMD products ?

**Comment by algotrash** [comment]
did you run it as administrator?

**Comment by JeffersonMontgomery-Intel** [comment]
No, PresentMon supports all GPUs.  We'll need more information to try and figure out what is going wrong:
- Are you running a target graphics application to measure?
- What PresentMon command line are you using?
- What OS are you using?
- Have you tried the latest version of PresentMon (https://github.com/GameTechDev/PresentMon/releases/latest)?
- Are you able to capture and provide an ETL for debugging (https://github.com/GameTechDev/PresentMon/blob/main/CONTRIBUTING.md#if-there-is-something-wrong-with-the-data-presentmon-is-reporting)?
